_episodes/02-starting-with-data.md

```diff
@@ -18,7 +18,11 @@ objectives:
     - "Perform basic mathematical operations and summary statistics on data in a Pandas DataFrame."
     - "Create simple plots."
 keypoints:
-    - "FIXME"
+    - "Libraries enable you to add functions to Python." 
+    - "Pandas is a popular library for working with data."
+    - "A DataFrame can be used to access data either by column name, column integer, or row."
```

Comment:
Dataframe is a Pandas data structure that allows one to access data by column (name or index) or row.

Reply:
```suggestion
    - "A Dataframe is a Pandas data structure that allows one to access data by column (name or index) or row."
```